test/Microsoft.AspNetCore.Server.KestrelTests/EngineTests.cs

```diff
@@ -301,11 +301,14 @@ public void ConnectionCanReadAndWrite(TestServiceContext testContext)
             {
                 using (var connection = new TestConnection(server.Port))
                 {
-                    await connection.Send(
```

Comment:
Testing hangs without this change, but also putting it in matches the other tests - so am ok with it :)